Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -1341,17 +1342,15 @@ script:
         }
     }
 
-    function memedump() {
-        if (commpress) {
-            args.commpress = args.commpress === 'yes';
-        }
-        let result = memdumpRequest(args['session'], args.path, args.commpress);
+    function memdump() {
+        let result = memdumpRequest(args['session'], args.path, args.compress);
         //wait for 1 second before trying to get command information
         sleep(1000);
         let timeout = (args['wait-timeout']) ? parseInt(args['wait-timeout']) * 1000 : DEFAULT_WAIT_TIMEOUT;
 
-        let command = getCommandInfo(args['session'], result.id, timeout, SLEEP_BETWEEN_RETRIES, args['cancel-on-timeout']);
-        return commandEntries['memedump'](command);
+        let command = getCommandInfo(args['session'], result.id, timeout, SLEEP_BETWEEN_RETRIES,
+        args['cancel-on-timeout']);
```

Comment:
Please put on the same line, this is JS and not python so no line length conventions exist

Reply:
done